{
  "ID": "pyup.io-25791",
  "Advisory": "django-social-auth3 0.7.2 fixes a security hole - redirects via the next param are now properly sanitized to disallow redirecting to  external hosts.",
  "Cve": "",
  "Specs": [
    "<0.7.2"
  ],
  "v": "<0.7.2"
}
